### ISSUE TITLE
About the categories of RSS Feed

### ISSUE THREAD

**Issue by starsareintherose** [yang991178/fluent-reader-lite#23]
**Is your feature request related to a problem? Please describe.**
In the section of feeds, could it can be organized by categories?
**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by yang991178** [comment]
Currently, you can filter by categories. Do you mean something folder-like as in the desktop app?

**Comment by starsareintherose** [comment]
Right, that will be great to allow categories. and in the feed page, there are no categories, only present by the time

**Comment by CubicrootXYZ** [comment]
Jumping on this issue instead of opening a new one. I'd really appreciate if there would be an option to filter for categories in the main screen. So I can only get unread items from a specific group. That could be added to the already existing filter menu.

**Comment by causa-prima** [comment]
I'll also jump in, describing my use case:
I do catch up on the daily stuff by category as well, i.e. I start by reading all the feeds categorized as "news", then usually my "job"-feeds, and end with the feeds categorized as "pictures" and "leisure". However, this is currently pretty cumbersome, as I have to first tap on "Subscriptions", then tap on "Groups", which is much too small and much too far up the left to reach for a right-handed person on a modern phone. Then, I have to tap the group name, and tap "All articles" (because I don't care about the source at that point, that's why I categorized the feeds).

If I could jump to all feeds from a given category more quickly, I would be very happy.
